backend/src/openarchiefbeheer/destruction/api/permissions.py

```diff
@@ -64,7 +74,9 @@ class CanReassignDestructionList(permissions.BasePermission):
     message = _("You are not allowed to reassign the destruction list.")
 
     def has_permission(self, request, view):
-        return request.user.has_perm("accounts.can_start_destruction")
+        return request.user.has_perm(
+            "accounts.can_start_destruction"
+        ) or request.user.has_perm("accounts.can_review_destruction")
```

Comment:
I guess that this is needed so that the main reviewer can add co-reviewers to the list? But this would also mean that the reviewer can unassign themselves and add someone else as main reviewer. We need to clarify if this is something that is desired :thinking:

Reply:
This is not needed to assign co-reviewers but to indeed unassign themselves, I left this out at first but then a test reminded me this apparently was supposed to work. I can't tell how exactly given this permission was already missing in main.

Reply:
I asked and for now: we remove the functionality and this is to be discussed.